connectomics/discfibers/ea_disctract.m

```diff
@@ -182,21 +201,49 @@ function calculate(obj)
+            % merged_pathways.mat
+            if obj.use_adjacency
+                if obj.multi_pathways == 1
+                    connectome_folder = [ea_getconnectomebase('dMRI_multitract'), obj.connectome];
+                    ADJ_connectome_path = [connectome_folder,filesep,'merged_pathways_ADJ.mat'];
+                    obj.ADJ = load(ADJ_connectome_path); % but we need to precompute (ourselves!)
+                else
+                    [connectome_folder,name,~] = fileparts(cfile);
+                    ADJ_connectome_path = [connectome_folder,filesep,name,'_ADJ.mat'];
+                end
+
+                try 
+                    obj.ADJ = load(ADJ_connectome_path);
+                catch
+                    disp('The adjacency matrix is missing for the chosen connectome')
+                    disp('Please, pre-compute it using ea_compute_adj_matrix.m and store the file in the connectome folder')
+                    obj.ADJ = false;
+                    obj.use_adjacency = false;
+                end
+            else
+                obj.ADJ = false;
+            end
+
+
+            switch obj.connectivity_type
+                case 2    % if PAM, then just extracts activation states from fiberActivation.mat
                     pamlist = ea_discfibers_getpams(obj);
-                    [fibsvalBin, fibsvalSum, fibsvalMean, fibsvalPeak, fibsval5Peak, fibcell, obj.connFiberInd] = ea_discfibers_calcvals_pam(pamlist, obj, cfile);
-                otherwise                    
-                    % this is not needed for OSS-DBS
+                    [fibsvalBin, fibsvalSum, fibsvalMean, fibsvalPeak, fibsval5Peak, fibcell, connFiberInd] = ea_discfibers_calcvals_pam(pamlist, obj, cfile);
+                    obj.results.(ea_conn2connid(obj.connectome)).('PAM_Ttest').fibsval = fibsvalBin;
+                    obj.results.(ea_conn2connid(obj.connectome)).connFiberInd_PAM = connFiberInd;
+                otherwise     % check fiber recruitment via intersection with VTA             
                     if isfield(obj.M,'pseudoM')
                         vatlist = obj.M.ROI.list;
                     else
                         vatlist = ea_discfibers_getvats(obj);
                     end
-                    [fibsvalBin, fibsvalSum, fibsvalMean, fibsvalPeak, fibsval5Peak, fibcell, obj.connFiberInd] = ea_discfibers_calcvals(vatlist, cfile, obj.calcthreshold);
+                    [fibsvalBin, fibsvalSum, fibsvalMean, fibsvalPeak, fibsval5Peak, fibcell,  connFiberInd] = ea_discfibers_calcvals(vatlist, cfile, obj.calcthreshold);
+                    obj.results.(ea_conn2connid(obj.connectome)).('VAT_Ttest').fibsval = fibsvalBin;
+                    obj.results.(ea_conn2connid(obj.connectome)).connFiberInd_VAT = connFiberInd; % old ff files do not have these data and will fail when using pathway atlases
             end
             
-            obj.results.(ea_conn2connid(obj.connectome)).('ttests').fibsval = fibsvalBin;
```

Comment:
@Kinway25 since we can already know the type of the results from `obj.connectivity_type`, do you think it's still necessary to use different field names here (`PAM_Ttest` and `VAT_Ttest`, `connFiberInd_PAM` and `connFiberInd_VAT`)? Or it's mainly for the case that both are calculated for the same patient?

Reply:
If it's necessary to use different field names. For compatibility consideration, I would think it might be better to still keep `ttests` and `connFiberInd`, but add new fields `ttests_PAM` and `connFiberInd_PAM` for the PAM `connectivity_type`.

Reply:
We should have PAM_Ttest and VAT_Ttest to switch between different connectivity models within one patient. I do not exclude some redundancy, but it also helped me to manage the code, and I do not think it affects the size of the files (of course, the files are larger if we have both connectivity models)

TBH, I am not sure we should even use the word  'ttest' when it comes to connectivity, since those are two separate notions.

Reply:
size is not a problem. I was more thinking about the compatibility with old analyses if we change the field name now.

Reply:
It is compatible, that we checked. We set the connectivity type to 1 (VTA) by default

Reply:
you mean even there's no `tractset.results.CONN.VAT_Ttest` in the old analysis, it still works? or the old field name `ttests` is now automatically renamed when loading?

Reply:
```
                %  if old fibfilt is used, assume it was computed with VATs
                if ~isfield(app.tractset.results.(ea_conn2connid(app.tractset.connectome)), 'VAT_Ttest') && ~isfield(app.tractset.results.(ea_conn2connid(app.tractset.connectome)),'PAM_Ttest')
                    app.tractset.results.(ea_conn2connid(app.tractset.connectome)).('VAT_Ttest') = app.tractset.results.(ea_conn2connid(app.tractset.connectome)).('ttests');
                end

                %if app.tractset.results.(ea_conn2connid(app.tractset.connectome)).connFiberInd
                %    app.tractset.results.(ea_conn2connid(app.tractset.connectome)).connFiberInd_VAT = app.tractset.results.(ea_conn2connid(app.tractset.connectome)).connFiberInd;
                %end
```
we have the above there. `connFiberInd` part was commented out? `ttests` field should be removed after setting the new field?

Reply:
if ~isfield(app.tractset.results.(ea_conn2connid(app.tractset.connectome)), 'VAT_Ttest') && ~isfield(app.tractset.results.(ea_conn2connid(app.tractset.connectome)),'PAM_Ttest')
                    app.tractset.results.(ea_conn2connid(app.tractset.connectome)).('VAT_Ttest') = app.tractset.results.(ea_conn2connid(app.tractset.connectome)).('ttests');
                end